index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <!-- https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP -->
     <meta http-equiv="Content-Security-Policy" content="default-src 'self'; script-src 'self'">
+    <link href="./styles.css" type="text/css" rel="stylesheet" />
```

Comment:
[It's recommended to omit `type` for CSS](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-type):

```suggestion
    <link href="./styles.css" rel="stylesheet">
```

Also removed the trailing ` /` for consistency with the other tags without ending pairs, like `<meta>`.